### DISCUSSION TITLE
why the Euler angle shown in MTEX is different from that in Channel5?

### DISCUSSION THREAD

**Discussion by 1996phd** [mtex-toolbox/mtex#1267] [Ask Anything] [answered]
![image](https://user-images.githubusercontent.com/93564255/139778698-469a988d-4c78-421e-b2c1-72e02143761e.png)
![image](https://user-images.githubusercontent.com/93564255/139778715-efccf9c6-39ac-43db-b3e8-f17d962ed6e5.png)
As shown in the two figures, the first one is from Channel5 and the second from MTEX, I'm wondering why the Euler angle of the same site is different

**Comment by zmichels** [comment]
If I had to guess.. maybe different reference frame(?) and/or Euler convention(?).

**Reply by 1996phd** [reply]
And I notice that euler angle in Channel5 could not exceed 180°, though they are both in "ZXZ" convention

**Comment by kilir** [comment, marked as answer]
Hi,
in general, have a look here https://mtex-toolbox.github.io/MTEXvsBungeConvention.html
The issue that Channel5 limits Euler angles to 180° has been answered here in the past. Try the search. Have a look here as well: https://mtex-toolbox.github.io/ODFPlot.html#4
Cheers,
Rüdiger

**Reply by 1996phd** [reply]
I appreciate your help.
